stream-log/src/main/kotlin/io/getstream/log/StreamLog.kt

```diff
@@ -36,45 +36,68 @@ import io.getstream.log.Priority.WARN
 public object StreamLog {
 
     /**
+     * Represent a [StreamLogger] is already installed or not.
      * Let you know if the internal StreamLogger instance used for logs has been initialized or it is using the
-     * default one
+     * default one.
      */
-    public var defaultLoggerOverridden: Boolean = false
+    @JvmStatic
+    public var isInstalled: Boolean = false
         private set
 
     /**
      * [StreamLogger] implementation to be used.
      */
+    @Volatile
     @PublishedApi
-    internal var internalLogger: StreamLogger = ErrorStreamLogger
+    internal var internalLogger: StreamLogger = SilentStreamLogger
```

Comment:
We want to have `ErrorStreamLogger` as the initial one.
It will show errors on the client apps even if they didn't initialize `StreamLog` yet.
It is needed on the case our SDK is used without init it properly (For example, Push or workers)

Reply:
Oh, I missed this by merging the PR 😢 I just rollback this in [91db533](https://github.com/GetStream/stream-log/pull/5/commits/91db533c5752af8ae27f12121659c44e2ac74d77).